Dockerfile

```diff
@@ -1,8 +1,19 @@
-FROM alpine:3.10
+FROM nixos/nix
```

Comment:
We should pin to a nix image, we did actually run into build issues a little while ago.